src/routers/secure/items.py

```
@@ -111,107 +129,113 @@ class ItemsResponse(BaseModel):
-            types = [type]
-        if "anime" in types:
-            types = [type for type in types if type != "anime"]
-            query = query.where(
-                or_(
+        media_types: Set[str] = {t.value for t in type}
+
+        if MediaTypeEnum.ANIME in type:
+            media_types.remove(MediaTypeEnum.ANIME.value)
+
+            if not media_types:
+                query = query.where(MediaItem.is_anime == True)
```

Ruff (0.14.1)

Avoid equality comparisons to `True`; use `MediaItem.is_anime:` for truth checks

Replace with `MediaItem.is_anime`

(E712)